src/cli_util.rs

```diff
@@ -276,6 +277,7 @@ impl CommandHelper {
         string_args: Vec<String>,
         global_args: GlobalArgs,
         settings: UserSettings,
+        layered_configs: LayeredConfigs,
```

Comment:
Clippy doesn't like this 8th argument. Could you suggest how to restructure those (or where to plumb through these layered_configs if not here).

It's partially redundant with UserSettings, but consolidating those got a little messy in the calling code.

Reply:
Feel free to tell Clippy to ignore it for now.